src/schoology-extractor/tests/data_generation/__main__.py

```diff
@@ -90,6 +102,61 @@
     logger.exception(ex)
 
 
+NUMBER_OF_ASSIGNMENTS_PER_SECTION = 2
+assignments = {}
+try:
+    assignments = generate_and_load_assignments(
+        request_client=request_client,
+        assignment_per_section_count=NUMBER_OF_ASSIGNMENTS_PER_SECTION,
+        enrollments=enrollments,
+    )
+except Exception as ex:
+    logger.exception(ex)
+
+
+NUMBER_OF_DISCUSSIONS_PER_ASSIGNMENT = 2
+discussions = {}
+try:
+    discussions = generate_and_load_discussions(
+        request_client=request_client,
+        discussions_per_assignment_count=NUMBER_OF_DISCUSSIONS_PER_ASSIGNMENT,
+        assignments=assignments,
+    )
+except Exception as ex:
+    logger.exception(ex)
+
+
+NUMBER_OF_DISCUSSION_COMMENTS_PER_DISCUSSION = 3
+discussion_comments = {}
+try:
+    discussion_comments = generate_and_load_discussion_comments(
+        request_client=request_client,
+        discussion_comments_per_discussion_count=NUMBER_OF_DISCUSSION_COMMENTS_PER_DISCUSSION,
+        discussions=discussions,
+        enrollments=enrollments,
+    )
+except Exception as ex:
+    logger.exception(ex)
+
+
+try:
+    rollback_loaded_discussion_comments(request_client, discussion_comments)
+except Exception as ex:
+    logger.exception(ex)
```

Comment:
When we're ready to do a real load of a large set of data, do you plan on just commenting out the rollbacks?

Reply:
Yes.  It's in the readme, but I'll edit to better call it out.